gitpod-ws.code-workspace

```diff
@@ -23,6 +23,7 @@
         { "path": "components/ws-daemon" },
         { "path": "components/ws-manager" },
         { "path": "components/ws-proxy" },
+        { "path": "components/public-api-server" },
```

Comment:
What's the effect of this? I suspect it's related to VS Code which I don't use so not sure how it behaves.

Reply:
It adds the module to the workspace, so that gopls understands it. We don't have any Golang tool support in VS Code otherwise.

Reply:
Thanks!